src/ui/components/Rating/index.js

```diff
@@ -68,9 +68,14 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: rating ? i18n.sprintf(i18n.gettext(`Update your rating to %(thisRating)s out of 5.`), { thisRating }) :
+          i18n.sprintf(i18n.gettext(`Rate this add-on %(thisRating)s out of 5.`), { thisRating }),
       };
 
       if (readOnly) {
+        props.title = rating ? i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5.'),
+          { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) }) :
+          i18n.gettext(`Unable to retrieve rating.`);
```

Comment:
Sorry for the drive-by review, but do you think a better message might be `This add-on has not been rated yet`? The `Unable` suggests (to me at least) that some sort of error occurred.

Reply:
Swapped out the error messages and updated the GIFs in the OP - thanks Bob!